src/App.js

```diff
@@ -1,13 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
-import Form from './components/Forms/Form';
```

Comment:
Is it ok if we change the "Components" file name to "components" with all lowercase? I think convention is to only capitalize the first letter of an actual component. Hope that makes sense.

Reply:
Yes! Just made that change, thanks for catching that!